definitions/checks/foreman_tasks/not_running.rb

```diff
@@ -6,17 +6,27 @@ class NotRunning < ForemanMaintain::Check
       tags :pre_upgrade
       after :foreman_tasks_not_paused
       before :check_old_foreman_tasks
+      param :wait_time, 'Time to wait in minutes for foreman tasks to finish'
```

Comment:
@patilsuraj767, we should avoid having a parameterized check except checks that are used internally within other definitions.